controllers/admin/AdminLegacyLayoutController.php

```diff
@@ -49,6 +49,7 @@ public function __construct($controllerName = '', $title = '', $headerToolbarBtn
         $this->enableSidebar = $enableSidebar;
         $this->helpLink = $helpLink;
         $this->php_self = $controllerName;
+        $this->className = 'LegacyLayout';
```

Comment:
Not sure this will not generate regressions 😢but without it, AdminLegacyLayoutController has an empty classname and this prevents some necessary variables from being injected into smarty context, such as `$context->link`

Reply:
what if you only set it in case "lite display" feature is used?

Reply:
Feels a bit like a hack 😅

Reply:
whole `AdminLegacyLayoutController` feels like a hack. :trollface:

Reply:
Yes indeed, but I dont want to create a situation where a property that looks harmless is set sometimes but not. That could create an headache to a future PS developer trying to debug something in there.